services/review_manager.js

```diff
@@ -17,3 +17,24 @@ exports.post_review_process = async(req,resp,input_response)=>{
   resp.send(input_response)
 }
 
+exports.pre_process_check_eligibility = async(req,resp)=>{
+  const eligible = await models.Enrollment.findOne({ where: { coach_id: req.query.coach_id,user_id:req.user.user_id } });
```

Comment:
move this logic to process_check method and return coach_id and user_id from  pre process